docs/DeveloperGuide.md

```diff
@@ -257,29 +257,61 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
+* Needs to manage a large number of contacts
+* Has a busy schedule with many appointments and meetings
+* Needs to know when he is free very quickly
+* May need documents or information related to contacts or meetings
+* Prefers keyboard over mouse interactions for on-the-go usage
+* Is reasonably comfortable using apps with text-based inputs
+* Currently uses excel for managing contacts
```

Comment:
I think can be a bit more broad here, esp since i'm gonna try to make import from google/outlook contacts

Reply:
Are you confident of making it happen because this is a baseline requirement rather than a 1-to-1 match to features per se.